IngresseSDK/Services/TransactionService.swift

```diff
@@ -33,4 +33,25 @@ public class TransactionService: BaseService {
             onError(error)
         }
     }
+    
+    public func getCheckinStatus(_ ticketCode: String, userToken: String, onSuccess: @escaping (_ checkinSession: [CheckinSession])->(), onError: @escaping (_ error: APIError) -> ()) {
+        
+        let url = URLBuilder(client: client)
+            .setPath("ticket/\(ticketCode.stringWithPercentEncoding()!)/status")
```

Comment:
create a var for ticket code With Percent Encoding, before putting it into url string path